bench/irmin-pack/trace_stat_summary_pp.ml

```diff
@@ -56,420 +56,294 @@ end
+        | `Play _ -> "n/a"
+        | `Replay s -> (
+            match s.path_conversion with
+            | `None -> "none"
+            | `V1 -> "v1"
+            | `V0_and_v1 -> "v0+v1"
+            | `V0 -> "v0"))
+
+  let box_of_summaries_config summary_names (summaries : summary list) =
+    let row0 =
+      if List.length summary_names = 1 then [] else [ "" :: summary_names ]
+    in
+    let rows =
+      List.map
+        (fun e ->
+          let n = name_of_summary_config_entry e in
+          let l = List.map (fun s -> cell_of_summary_config s e) summaries in
+          n :: l)
+        summary_config_entries
+    in
+    row0 @ rows |> Pb.matrix_to_text
+end
 
-  (* Step 2/3 - Prepare the functions to build all the simple floors *)
-  let zip : (summary -> curve) -> (string * curve) list =
-   fun curve_of_summary ->
-    List.map2
-      (fun sname s -> (sname, curve_of_summary s))
-      summary_names summaries
-  in
-  let zip_per_block_to_per_sec : (summary -> curve) -> (string * curve) list =
-    let sec_per_block =
+(** Highlights *)
+module Table1 = struct
+  let rows_of_summaries summaries =
+    let cpu_time_elapsed = List.map (fun s -> s.elapsed_cpu) summaries in
+    let add_per_sec =
       List.map
         (fun s ->
-          all_9_ops
-          |> List.map (fun op ->
-                 mul_curves Summary.(Op.Map.find op s.ops).duration.evolution
-                   Summary.(Op.Map.find op s.ops).count.evolution)
-          |> sum_curves)
+          fst Summary.(Op.Map.find `Add s.ops).cumu_count.max_value
+          /. s.elapsed_cpu)
         summaries
     in
-    fun curve_of_summary ->
-      List.map2
-        (fun (sname, sec_per_block) s ->
-          (sname, div_curves (curve_of_summary s) sec_per_block))
-        (List.combine summary_names sec_per_block)
+    let tail_latency =
+      List.map
+        (fun s -> fst Summary.(Op.Map.find `Commit s.ops).duration.max_value)
         summaries
-  in
-
-  let v : string -> (summary -> Summary.linear_bag_stat) -> summary_floor =
-   fun stat_name lbs_of_summary ->
-    let curves = zip (fun s -> (lbs_of_summary s).value.evolution) in
-    `Data (`R, stat_name, curves)
-  in
-  let pb : string -> (summary -> Summary.linear_bag_stat) -> summary_floor =
-   fun stat_name lbs_of_summary ->
-    let curves = zip (fun s -> (lbs_of_summary s).diff_per_block.evolution) in
-    `Data (`R, stat_name, curves)
-  in
-  let ps : string -> (summary -> Summary.linear_bag_stat) -> summary_floor =
-   fun stat_name lbs_of_summary ->
-    let curves =
-      zip_per_block_to_per_sec (fun s ->
-          (lbs_of_summary s).diff_per_block.evolution)
     in
-    `Data (`R, stat_name, curves)
-  in
-
-  let op_count : string -> Op.Key.t -> summary_floor =
-   fun name op ->
-    let curves =
-      zip (fun s -> Summary.(Op.Map.find op s.ops).count.evolution)
+    let tx_per_sec =
+      (* TODO: When replaying on a middle section of the chain, set
+         [~first_block_idx] *)
```

Comment:
Is this a feature we support?

Reply:
Not yet, but that's on the top of my TODO